contracts/FeralFileAirdropV1.sol

```diff
@@ -0,0 +1,133 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+import "./Authorizable.sol";
+
+contract FeralFileAirdropV1 is ERC1155, Authorizable {
+    enum Type {
+        Fungible,
+        NonFungible
+    }
+
+    // Version
+    string public constant version = "v1";
```

Comment:
```solidity
    // version code of contract
    string public constant codeVersion = "FeralfileAirdropV1";
```

Reply:
This is a new contract dedicated for the activation so i don't follow what we did for the exhibition contract. I think the `version` is more common to describe contract version and the `v1` is good enough as well.

Reply:
but we use that column to check the version at ff backend, missing that will make ff not able to identify this contract